### ISSUE TITLE
Plugin does not report error, but flow-bin does (for npm imported types)

### ISSUE THREAD

**Issue by jbinto** [amilajack/eslint-plugin-flowtype-errors#62]
This plugin seems to have issues detecting/reporting on flow errors when they span across multiple files/npm projects.

For example:

```js
import { GraphQLObjectType } from 'graphql'
import type { GraphQLOutputType } from 'graphql'

// [...]

// The following line (12) is detected as a flow error by the plugin and flow cli
const x: number = "lol"

// The following line (15) is detected as a flow error by the flow CLI, but not the plugin
const y: GraphQLOutputType = new GraphQLObjectType({
  name: 'foo'
  // fields: (missing, flow should complain)
})
```

Output of `flow-bin` (detecting both errors):

```
src/index.js:12
 12: const x: number = "lol"
                       ^^^^^ string. This type is incompatible with
 12: const x: number = "lol"
              ^^^^^^ number

src/index.js:15
 15: const y: GraphQLOutputType = new GraphQLObjectType({
                                  ^ constructor call
383:   constructor(config: GraphQLObjectTypeConfig<*, *>) {
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ property `fields`. Property not found in. See: node_modules/graphql/type/definition.js.flow:383
 15: const y: GraphQLOutputType = new GraphQLObjectType({
                                                        ^ object literal


Found 2 errors
```

Output of `eslint` (not detecting the 2nd error):

```
/Users/jbuchanan/dev/tb/foo/src/index.js
  12:20  error  string:  This type is incompatible with 'number'. See line 12  flowtype-errors/show-errors

✖ 1 problem (1 error, 0 warnings)
```

Perhaps the hint from flow to "see node_modules/graphql...:383" might be throwing us off? 

I have a reproduction case here, if it helps (fairly simple repo created with `create-react-app`), this commit has all you need).

https://github.com/jbinto/flowtype-errors-plugin-issue-npm/commit/3dcbcd41a67ba642954fd8ea37d30bb286013540

**Comment by jbinto** [comment]
Also here is a gist containing the output of `flow check --json`, which should help a little more:

https://gist.githubusercontent.com/jbinto/81ecf6ba259596ef4234aa0ff7a79951/raw/72a43100f885a57c8c9a48421d859c75c4c4bcd0/flow-output.json

**Comment by jbinto** [comment]
Seems related to #33.

**Comment by amilajack** [comment]
Sorry for very late response. Yes, this is indeed related to #33 I'll look into it this weekend. Would be great if you could also look into this and submit a PR :)
